components/lsaHJAS68dskjHfslkdj2.yml

```diff
@@ -0,0 +1,25 @@
+name: emotion_response_selector
+display_name: Emotion Ranking-based Response Selector
+component_type: null
+model_type: Dictionary/Pattern-based
+is_customizable: false
+author: publisher@deeppavlov.ai
+description: Algorithm that selects the final emotional response among the given list of candidate
+    responses via the given ranking service
+ram_usage: 100M
+gpu_usage: null
+group: response_selectors
+connector:
+  protocol: http
+  timeout: 1.0
+  url: http://emotion-ranking-based-response-selector:8048/respond
+dialog_formatter: state_formatters.dp_formatters:cropped_dialog
+response_formatter: state_formatters.dp_formatters:base_response_selector_formatter_service
+previous_services:
+- candidate_annotators
+required_previous_services: null
+state_manager_method: add_bot_utterance
+tags: null
```

Comment:
selectors. And in pipeline conf please also fix it. As in skill selector (this is a bug in whole dream -- I am goijng to fix it in a separate pull request for all response selectors. I actually thought I did it previously, but seems like not yet).
https://github.com/deeppavlov/dream/blob/dev/assistant_dists/dream_persona_openai_prompted/pipeline_conf.json#L159-L161 (also find a card for this component and make analogy)

Reply:
added tags to response selector in `component card` and in `pipeline_conf`